doc/packages/krita.section.md

```diff
@@ -0,0 +1,27 @@
+# Krita {#sec-krita}
+
+## Installing binary plugins {#krita-install-plugins}
+
+You can choose what plugins are added to Krita by overriding the
+`plugins` attribute.
```

Comment:
I'm fine with this new `plugins` attribute, but maybe it is an overkill? I'm not aware of anything "native plugin" other than GMIC. It might be simpler if we always build krita with GMIC plugin

Reply:
In the past there where 0 external native plugins, now there is 1. 
It seems sane to preemptively add support for more if it's trivial to do, no?

Reply:
> It seems sane to preemptively add support for more if it's trivial to do

Probably no one has a plan to introduce new "native plugin", and I don't think this "native plugin" thing is not designed to be used to extend Krita (I might be wrong). Documenting how to add additional plugins while no additional plugin exists might confuse people.